app/views/components/gds/linkWithHiddenValue.scala.html

```diff
@@ -0,0 +1,26 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.govukfrontend.views.html.components.FormWithCSRF
+
+@this(formHelper: FormWithCSRF)
+
+@(text: String, call: Call, value: String)(implicit request: Request[_], messages: Messages)
+
+@formHelper(action = call, 'autoComplete -> "off") {
+  <input type="hidden" name="value" value="@value" />
+  <input type="submit" class="input-submit govuk-link govuk-link--no-visited-state" value="@text"/>
+}
```

Comment:
Yeah, post our conversation after stand up, lets drop this idea and stick to showing the value in the link :+1: